Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```
@@ -0,0 +1,1107 @@
+category: Authentication
+commonfields:
+  id: Thycotic
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'true'
+  display: Turns on synchronization of secrets with XSOAR credential store.
+  hidden: false
+  name: isFetchCredentials
+  required: false
+  type: 8
+- display: A list secret names to retrieve and sync (separated by commas). Note the
+    sync, will function only if the synchronization check box is selected.
+  hidden: false
+  name: credentialobjects
+  required: false
+  type: 12
+description: Secret Server is the only fully featured Privileged Account Management
+  (PAM) solution available both on premise and in the cloud. It empowers security
+  and IT ops teams to secure and manage all types of privileged accounts and offers
+  the fastest time to value of any PAM solution.
+display: Thycotic
+name: Thycotic
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieved password from secret
+    execution: false
+    name: thycotic-secret-password-get
+    outputs:
+    - contextPath: Thycotic.Secret.Password
+      description: Current password from secret
+      type: String
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieved username from secret
+    execution: false
+    name: thycotic-secret-username-get
+    outputs:
+    - contextPath: Thycotic.Secret.Username
+      description: Retrieves username from secret.
+      type: String
+  - arguments:
+    - default: false
+      description: Search for secret using secret name.
+      isArray: false
+      name: search_name
+      required: true
+      secret: false
+    deprecated: false
+    description: Search ID secret by Name
+    execution: false
+    name: thycotic-secret-search-name
+    outputs:
+    - contextPath: Thycotic.Secret.Id
+      description: Retrieves list of Secret IDs.
```

I also suggest rephrasing the description of context outputs, here and on other outputs as well, from a verb form to a noun description describing the essence of the output. See suggestion:
```suggestion
      description: Secret ID.
```

---

Done